tao_compiler/mlir/disc/tests/regression/kstitch_fusion.cc

```diff
@@ -133,4 +133,19 @@ TEST(KStitchFusionGPUTest, KStitch3DF64) {
   unsetenv("DISC_EXPECTED_KERNELS_IN_UT");
 }
 
-}  // namespace mlir_test
\ No newline at end of file
+// With splat const in the kStitch fusion
+TEST(KStitchFusionGPUTest, KStitchSimpleWithSplatConstF32) {
+  setenv("DISC_ENABLE_STITCH", "true", 1);
+  setenv("DISC_EXPECTED_KERNELS_IN_UT", "1", 1);
+  EXPECT_TRUE(feature_test_main(
+      /*mlir_file_path*/ c_ft_path + "kstitch_fusion_with_splat_const.mlir",
```

Comment:
nit. Seems not a kstitch fusion. May rename it in the next PR?

Reply:
it IS a kStitch kernel since Max is a reduce operation.